composer.json

```
@@ -15,28 +15,28 @@
         }
     ],
     "require": {
-        "php": "^7.4 || ^8.0",
+        "php": ">=8.0.2",
         "symfony-cmf/routing": "^2.3.2",
```

```suggestion
        "symfony-cmf/routing": "dev-master as 3.0.0",
```

https://github.com/symfony-cmf/Routing/compare/2.3.4...master - we started on this but did not release yet.

probably need to go over the code to see if there are other cleanups - but would be good to see the build with the bundle as well, to prepare the release of both together

---

Thanks, that's been updated